docker-compose-production.yml

```diff
@@ -0,0 +1,24 @@
+# ./docker-compose-production.yml
+version: '3'
+services:
+    app:
+        container_name: hicsail
+        image: hicsail/emotion-map-backend
+        restart: always
+        ports:
+            - 3000:3000
+        env_file:
+            - ./stack.env
+        depends_on:
+            db:
+                condition: service_healthy
+    
+    db:
+        image: mysql:8.0
+        restart: always
+        volumes:
+            - ./db:/docker-entrypoint-initdb.d
```

Comment:
We want to use a volume instead of the FS here:

Example: https://github.com/hicsail/authentication-service/blob/main/deployment/docker-compose-production.yml#L27C1-L46C21